docs/eventsApi.js.html

```diff
@@ -0,0 +1,193 @@
+
```

Comment:
❔ This file appears to be generated by JSDoc.  Typically, generated files aren't included in the repository, since humans shouldn't make changes to them (their changes would just get overwritten the next time someone re-runs the command to generate the documentation).  Maybe consider removing these files and adding some extra lines to `.gitignore`.

Reply:
So where do the up to date generated docs live? Do they have their own repo? Or do you only generate them locally?....sorry experiencing some confusion!!

Reply:
A common strategy is to have the CI/CD pipeline generate the files and deploy them somewhere (e.g., modify the Github action to deploy the documentation to Github Pages).

It's similar in concept to writing code that has to be compiled before it can be deployed.  E.g., to deploy a Java webapp you have to compile the Java code files into a `.war` archive, but rather than committing the `.war` file, you instead commit the Java code files into the repo, and then CI/CD runs the `make` command to build the archive and deploy it.

In this case, the "code" that gets deployed is JSDoc instead of Javascript, but it's the same idea 😺